kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -366,17 +374,19 @@ public String getClusters()
     return Joiner.on(';').join(clusters);
   }
 
-  public void setTopics(String... topics)
+  public void setTopics(String topics)
   {
-    this.topics = topics;
+    Topic.validate(topics.replace(" |,", "");
```

Comment:
valid topic + valid topic is valid  invalid topic + valid topic is invalid

I know there is one exception here "a b, c" will be treated as valid but this is just a fast fail. A topic contain space in the middle will eventually throw exception.
To keep it simple I did this quick shortcut validation.
And there is no false negative